pageserver/src/tenant/timeline/walreceiver/connection_manager.rs

```diff
@@ -421,7 +421,11 @@ impl ConnectionManagerState {
                     Err(e) => {
                         use super::walreceiver_connection::ExpectedError;
                         if e.is_expected() {
-                            info!("walreceiver connection handling ended: {e:#}");
+                            if let Some(adjusted_error) = e.maybe_reformat() {
+                                info!("walreceiver connection handling ended: {adjusted_error}");
+                            } else {
+                                info!("walreceiver connection handling ended: {e}")
```

Comment:
`{e:#}` for "display [with the] alternate [bit]".

You should familiaze yourself with the differences between:
- `println!("{e}", &anyhow::Error::new("foobar").context("outer context"))`
- `println!("{e:#}", &anyhow::Error::new("foobar").context("outer context"))`
- `println!("{e:?}", &anyhow::Error::new("foobar").context("outer context"))`

Reply:
TIL.  As it turned out, in latest version this goes away because the error print is taking a postgres error directly rather than an `anyhow` error.